tools/sdk/include/crypto/bigint.h

```diff
@@ -0,0 +1,99 @@
+/*
```

Comment:
Do you think there is a way to submodule these instead? If anything, i have one more use case for using axTLS as a submodule, to enable root certificate store (https://github.com/igrr/axtls-8266/issues/44).